### PR TITLE
Make junit analysis optional

### PR DESCRIPTION
**Pull request by dobbymoodge**
Adds `junit_analysis` job config flag, defaults to `True`.

Set `junit_analysis` to `False` in `test_pull_request_origin_integration.yml` overrides

### CODE REVIEW

File: sjb/templates/test_case.xml [hunk 1]
```diff
@@ -126,12 +126,14 @@ fi
       <defaultExcludes>true</defaultExcludes>
       <caseSensitive>true</caseSensitive>
     </hudson.tasks.ArtifactArchiver>
```
Comment (dobbymoodge) [inline sjb/templates/test_case.xml]: @stevekuznetsov Should I also make the `<artifacts>` value exclude the junit artifacts if `junit_analysis` is false?

Comment (stevekuznetsov) [reply sjb/templates/test_case.xml]: You don't have to

**Comment by stevekuznetsov** [comment]
Can you update the doc at `sjb/syntax.md`

**Comment by dobbymoodge** [comment]
@stevekuznetsov updated docs

File: sjb/syntax.md [hunk 2]
```diff
@@ -86,6 +86,15 @@ fails. The syntax is:
 email: [] # list of e-mail addresses
 ```
 
+## `junit_analysis`
+
+`junit_analysis` is an optional flag to enable or sdisable the "Publish
```
Comment (stevekuznetsov) [inline sjb/syntax.md]: spelling

Comment (dobbymoodge) [reply sjb/syntax.md]: done

File: sjb/config/test_cases/test_pull_request_origin_integration.yml [hunk 3]
```diff
@@ -3,4 +3,5 @@ parent: 'test_cases/test_branch_origin_integration.yml'
 overrides:
   sync_repos:
     - name: "origin"
-      type: "pull_request"
\ No newline at end of file
+      type: "pull_request"
+  junit_analysis: False
```
[the remark this answers is not in the document]
Comment (dobbymoodge) [reply sjb/config/test_cases/test_pull_request_origin_integration.yml]: done

[the remark this answers is not in the document]
Comment (kargakis) [reply sjb/config/test_cases/test_pull_request_origin_integration.yml]: Why did we do this for this job?

**Pull request merged**